src/R/Components/Impl/PackageManager/Implementation/View/PackageDetails.xaml

```diff
@@ -2,11 +2,205 @@
              xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
              xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
              xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
-             xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             xmlns:wpf="clr-namespace:Microsoft.R.Wpf;assembly=Microsoft.R.Wpf"
              xmlns:view="clr-namespace:Microsoft.R.Components.PackageManager.Implementation.View"
-             mc:Ignorable="d" 
-             d:DesignHeight="300" d:DesignWidth="300">
-    <Grid>
+             xmlns:local="clr-namespace:Microsoft.R.Components"
+             xmlns:designTime="clr-namespace:Microsoft.R.Components.PackageManager.Implementation.View.DesignTime"
+             Background="{DynamicResource {x:Static wpf:Brushes.DetailPaneBackground}}"
+             Foreground="{DynamicResource {x:Static wpf:Brushes.UIText}}"
+             x:Name="Self"
+             mc:Ignorable="d"
+             d:DesignHeight="800" d:DesignWidth="400"
+             d:DataContext="{d:DesignInstance Type=designTime:DesignTimeRPackageViewModel, IsDesignTimeCreatable=True}">
+
+    <UserControl.Resources>
+        <ResourceDictionary>
+            <ResourceDictionary.MergedDictionaries>
+                <view:PackageManagerResources/>
+            </ResourceDictionary.MergedDictionaries>
+        </ResourceDictionary>
+    </UserControl.Resources>
+    <ScrollViewer x:Name="Root" HorizontalScrollBarVisibility="Disabled">
+        <Grid Margin="24,0,7,0">
+            <Grid.RowDefinitions>
+                <RowDefinition Height="Auto" />
+                <RowDefinition Height="Auto" />
+                <RowDefinition Height="Auto" />
+                <RowDefinition Height="Auto" />
+                <RowDefinition Height="Auto" />
+                <RowDefinition Height="Auto" />
+            </Grid.RowDefinitions>
+
+            <!-- Icon and Id -->
+            <StackPanel Grid.Row="0" Orientation="Horizontal" MinHeight="32" Margin="0,8">
+
+                <TextBox Background="Transparent" Foreground="{DynamicResource {x:Static wpf:Brushes.UIText}}" BorderThickness="0" Margin="-2,0,-2,0"
```

Comment:
Side note regarding screenshot - the disclaimer at the bottom should probably be saying "Microsoft is not responsible", since that's the legal entity.